lib/matterhorn/serialization/scoped_collection_serializer.rb

```diff
@@ -9,7 +9,20 @@ def serializable_hash
         super.merge!(TOP_LEVEL_KEY => serialized_object)
       end
 
+      def request_env
```

Comment:
this is part of the base class, https://github.com/blakechambers/matterhorn/blob/ordering/lib/matterhorn/serialization/scoped.rb#L50